### PR TITLE
Blog post on managing HE hosts

### PR DESCRIPTION
**Pull request by rgolangh**
This is a blog post about the new 4.0 functionality of adding/removing HE hosts using the engine.

### CODE REVIEW

**Comment by rgolangh** [comment]
Abandoning. I used that as an example

**Comment by rgolangh** [comment]
- moved the image under source/images
- added some headings

File: source/blog/Manage-Your-Hosted-Engine-Hosts-Deployment.html.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Manage Your Hosted Engine Hosts Deployment Via Engine
+author: Roy Golan rgolan@redhat.com
+tags: hosted-engine, deployment, infrastructure
+date: Wed Jun 15 11:05:24 IDT 2016
+---
+
+# Managing your Hosted Engine hosts deployment via engine
+Hosted engine seen a lot of progress and evolution till today is the defacto recommended way to to deploy your engine. But since that special Hoseted Engine High Availability cluster itself needs management(we call it HA cluster), we worked on making that be managed as well by the (hosetd) engine itself.
```
Comment (rmohr) [inline source/blog/Manage-Your-Hosted-Engine-Hosts-Deployment.html.md]: s/hosetd/hosted/

File: source/blog/Manage-Your-Hosted-Engine-Hosts-Deployment.html.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Recent oVirt versions made it lot more easier to deploy hosted-engine, first by introducing the appliance and cloud-init customization
```
Comment (rmohr) [inline source/blog/Manage-Your-Hosted-Engine-Hosts-Deployment.html.md]: is there an 'a' missing? made it 'a' lot more easier

**Comment by bproffitt** [comment]
Okay, following up on this... is it ready to post?

**Comment by jzb** [comment]
@rgolangh Is this OK to post??

**Comment by duck-rh** [comment]
Please rename the blog post filename to follow the naming scheme (with date).

Also please rebase your branch btw.

**Comment by thatdocslady** [comment]
@rgolangh can you please implement duck's feedback and update when the post is ready?

**Pull request merged**